source/core.py

```diff
@@ -303,6 +303,9 @@ def main():
 	import speech
 	log.debug("Initializing speech")
 	speech.initialize()
+	from speech import sayAll
+	log.debug("Initializing sayAllHandler")
+	sayAll.initalize(sayAll.SpeechWithoutPauses(speakFunc=speech.speak))

```

Comment:
I think it would also be fine to construct the `SpeechWithoutPauses` inside of the initialize method.